src/libavrdude.h

```diff
@@ -1161,6 +1159,11 @@ int str_eq(const char *str1, const char *str2);
 int str_contains(const char *str, const char *substr);
 int str_ends(const char *str, const char *ends);
 
```

Comment:
Since all `str*` functions "belongs together", maybe you can remove the blank like here?  Then it looks more like these belong together, which they absolutely do; they are all string-checking functions.